frontend/app/documents/FinishOnboarding.tsx

```
@@ -0,0 +1,267 @@
+          </form>
+        </Form>
+      </DialogContent>
+    </Dialog>
+  );
+};
+
+const OnboardingCompleteModal = ({ open, onNext }: OnboardingStepProps) => {
+  const company = useCurrentCompany();
+
+  return (
+    <Dialog open={open} onOpenChange={() => {}}>
```

ESLint

[error] Unexpected empty arrow function.

(@typescript-eslint/no-empty-function)